pym/bob/cmds/build.py

```diff
@@ -514,10 +524,21 @@ def cook(self, steps, parentPackage, checkoutOnly, depth=0):
                 print(">>", colorize(self.__currentPackage, "32;1"))
 
     def _cookCheckoutStep(self, checkoutStep, depth):
+        overrides = []
+        scmList = checkoutStep.getScmList()
+        for scm in scmList:
+            for o in scm.getActiveOverrides():
+                if not o in overrides:
+                    overrides.append(o)
+                if not o in self.__statistic['activeOverrides']:
+                    self.__statistic['activeOverrides'].append(o)
+        overrides = len(overrides)
+
         checkoutDigest = checkoutStep.getVariantId()
         if self._wasAlreadyRun(checkoutStep):
             prettySrcPath = checkoutStep.getWorkspacePath()
-            self._info("   CHECKOUT  skipped (reuse {})".format(prettySrcPath))
+            self._info("   CHECKOUT  skipped (reuse {})".format(prettySrcPath)
+                    + ((" (" + str(overrides) + " scm " + ("overrides" if overrides > 1 else "override") +")") if overrides else ""))
```

Comment:
The override string should be calculated once at the beginning and just used here and below...